datadog_checks_dev/datadog_checks/dev/tooling/utils.py

```diff
@@ -409,6 +408,37 @@ def get_metric_sources():
     return {path for path in os.listdir(get_root()) if file_exists(get_metadata_file(path))}
 
 
+def get_available_logs_integrations():
+    # List of integrations where is not possible or it does not make sense to have its own log integration
+    not_possible = {
```

Comment:
Small nit, but I feel like it would be better to keep this list outside as a constant, so that it would be easier to find if we add any more integrations to the list.